src/Catty/Views/Custom/CollectionViewCells/BrickCells/Pen/PenClearBrick+Instruction.swift

```diff
@@ -32,6 +32,7 @@ extension PenClearBrick: CBInstructionProtocol {
             else { fatalError("This should never happen!") }
 
         return {
+            spriteNode.penConfiguration.previousPosition = spriteNode.position
```

Comment:
I think there is no need to do this at the _PenClearBrick_, but I think it might be a very good idea to _reset_ the position at the _PenUpBrick_.